### PR TITLE
React Native Release 2.2.0

### PR DESCRIPTION
**Pull request by charmian01**
## Description
Bumped the versions
Updated TestApp to use latest SDKs
Updated the Changelog.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
 
 ### App Center
 
 * **[Feature]** Support React Native 0.60.0.
 
+#### Android
+
+* **[Fix]** Remove unsecure UUID fallback when UUID generation theorically fails, in reality it never fails.
+* **[Fix]** Check for running in App Center Test will now work when using AndroidX instead of the support library.
+* **[Feature]** Add `AppCenter.isRunningInAppCenterTestCloud` to provide method to check if the application is running in Test Cloud.
```
Comment (MatkovIvan) [inline CHANGELOG.md]: I guess this API isn't exposed to RN

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+
+#### iOS
+
+* **[Feature]** Add `isRunningInAppCenterTestCloud` in `MSAppCenter` to provide method to check if the application is running in Test Cloud.
```
Comment (MatkovIvan) [inline CHANGELOG.md]: The same

File: TestApp/ios/Podfile.lock [hunk 3]
```diff
@@ -264,4 +264,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: aa799989050601e16f05421c7ef07e0f979eec5f
 
-COCOAPODS: 1.7.5
+COCOAPODS: 1.7.4
```
Comment (MatkovIvan) [inline TestApp/ios/Podfile.lock]: Why do we downgrade it?

Comment (charmian01) [reply TestApp/ios/Podfile.lock]: Upgrade it in new commit.

File: CHANGELOG.md (hunk 1 above)
Comment (charmian01) [reply CHANGELOG.md]: Removed it.

File: CHANGELOG.md (hunk 2 above)
Comment (charmian01) [reply CHANGELOG.md]: Removed it.

**Pull request merged**